pyiceberg/io/fsspec.py

```diff
@@ -163,6 +164,9 @@ def _s3(properties: Properties) -> AbstractFileSystem:
 
     if request_timeout := properties.get(S3_REQUEST_TIMEOUT):
         config_kwargs["read_timeout"] = float(request_timeout)
+    
+    if s3_anonymous := properties.get(S3_ANONYMOUS):
+        config_kwargs["anon"] = bool(s3_anonymous)
```

Comment:
For consistency, I would recommend `strtobool` in `pyiceberg/types.py`:
```suggestion
        config_kwargs["anon"] = strtobool(s3_anonymous)
```

Reply:
ah, missed that helper, updated